src/main/java/eggy/command/FindCommand.java

```diff
@@ -0,0 +1,43 @@
+package eggy.command;
+
+import eggy.exception.IncompleteTaskException;
+import eggy.storage.Storage;
+import eggy.task.TaskList;
+import eggy.task.Task;
+import eggy.ui.Ui;
```

Comment:
I like that imports are explicitly named and clearly show what's used in the class!